python/mxnet/recordio.py

```diff
@@ -37,7 +37,7 @@ class MXRecordIO(object):
     """Reads/writes `RecordIO` data format, supporting sequential read and write.
 
     Example usage:
-    ----------
+    --------------
```

Comment:
Please change to: Examples,

Example usage is not supported numpy section title.

Reply:
fixed